### PR TITLE
modelo 390

### PR DESCRIPTION
**Pull request by casp-odoo**
wip

wip + fix oco

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_es/data/account_tax_data.xml [hunk 1]
```diff
@@ -75,6 +75,11 @@
         <field name="applicability">taxes</field>
         <field name="country_id" ref="base.es"/>
     </record>
+    <record id="mod_303_10_4_percent" model="account.account.tag">
+        <field name="name">mod303[10] - 4%</field>
+        <field name="applicability">taxes</field>
+        <field name="country_id" ref="base.es"/>
+    </record>
```
Comment (oco-odoo) [inline addons/l10n_es/data/account_tax_data.xml:82]: Useful ?

File: addons/account/models/account_report.py [hunk 2]
```diff
@@ -444,6 +444,8 @@ def create(self, vals_list):
         # with engine 'tax_tags'.
         for vals in vals_list:
             self._strip_formula(vals)
+            if 'formula' in vals and isinstance(vals['formula'], str):
+                vals['formula'] = vals['formula'].replace('\n', '').strip()
```
Comment (oco-odoo) [inline addons/account/models/account_report.py:448]: Shouldn't be there anymore; rebase ^^

File: addons/l10n_es/data/account_tax_data.xml [hunk 3]
```diff
@@ -369,12 +654,12 @@
...
+                'tag_ids': [ref('mod_303_28'), ref('mod_390_605')],
             }),
             (0,0, {
                 'repartition_type': 'tax',
                 'account_id': ref('l10n_es.account_common_472'),
-                'tag_ids': [ref('mod_303_29')],
+                'tag_ids': [ref('mod_303_29'), ref('mod_390_606')],
             }),
          ]"/>
         <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
```
Comment (oco-odoo) [inline addons/l10n_es/data/account_tax_data.xml:665]: No refund repartition ? (below as well)

**Review by oco-odoo — CHANGES_REQUESTED** [review]

File: addons/l10n_ch/demo/demo_company.xml [hunk 4]
```diff
@@ -24,7 +24,7 @@
 
     <function model="res.users" name="write">
         <value eval="[ref('base.user_root'), ref('base.user_admin'), ref('base.user_demo')]"/>
-        <value eval="{'company_ids': [(4, ref('l10n_ch.demo_company_ch'))]}"/>
+        <value eval="{'company_ids': [(4, ref('l10n_ch.demo>>_company_ch'))]}"/>
```
Comment (h4818) [inline addons/l10n_ch/demo/demo_company.xml:27]: 🤔

Comment (casp-odoo) [reply addons/l10n_ch/demo/demo_company.xml:27]: please ignore this PR, I just pushed so I would be sure to have my latest modification and be able to see them from the github interface, but I'll need to start it clean anyway :)

**Pull request closed without merging**